includes/admin/extensions/product-education/class-wpsmtp.php

```diff
@@ -0,0 +1,228 @@
+	private $manager;
+
+	public function __construct() {
+		add_filter( 'edd_settings_emails', array( $this, 'register_setting' ) );
+		add_action( 'edd_wpsmtp', array( $this, 'settings_field' ) );
+
+		$this->manager = new \EDD\Admin\Extensions\Extension_Manager();
+	}
+
+	/**
+	 * Register the setting to show the WP SMTP installer if it isn't active.
+	 *
+	 * @param array $settings
+	 * @return array
+	 */
+	public function register_setting( $settings ) {
+		if ( $this->is_smtp_configured() ) {
+			return $settings;
+		}
+		$settings['main']['wpsmtp'] = array(
+			'id'   => 'wpsmtp',
+			'name' => __( 'Improve Email Deliverability', 'easy-digital-downloads' ),
+			'desc' => '',
+			'type' => 'hook',
+		);
+
+		return $settings;
+	}
+
+	/**
+	 * Output the settings field (installation helper).
+	 *
+	 * @param array $args
+	 * @return void
+	 */
+	public function settings_field( $args ) {
+		$this->manager->enqueue();
+		?>
+		<div class="edd-extension-manager__body">
+			<p class="edd-extension-manager__description">
+				<?php esc_html_e( 'WP Mail SMTP allows you to easily set up WordPress to use a trusted provider to reliably send emails, including sales notifications.', 'easy-digital-downloads' ); ?>
+			</p>
+
+			<div class="edd-extension-manager__group">
+				<div class="edd-extension-manager__step">
+					<?php $this->manager->button( $this->get_button_parameters() ); ?>
+				</div>
+
+				<?php
+				if ( $this->is_smtp_activated() ) {
+					return;
+				}
+				?>
+				<div class="edd-extension-manager__step" style="display:none;">
+					<?php $this->manager->link( $this->get_link_parameters() ); ?>
+				</div>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Gets the button parameters.
+	 *
+	 * @return array
+	 */
+	private function get_button_parameters() {
+		$button = array();
+		// If neither the lite nor pro plugin is installed, the button will prompt to install and activate the lite plugin.
+		if ( ! $this->manager->is_plugin_installed( $this->config['lite_plugin'] ) && ! $this->manager->is_plugin_installed( $this->config['pro_plugin'] ) ) {
+			$button['plugin']      = $this->config['lite_download_url'];
+			$button['action']      = 'install';
+			$button['button_text'] = __( 'Install & Activate WP Mail SMTP', 'easy-digital-downloads' );
+		} elseif ( ! $this->is_smtp_activated() ) {
+			// If one of the SMTP plugins is installed, but not activated, the button will prompt to activate it.
+			$button['plugin']      = $this->config['lite_plugin'];
+			$button['action']      = 'activate';
+			$button['button_text'] = __( 'Activate WP Mail SMTP', 'easy-digital-downloads' );
+		} elseif ( ! $this->is_smtp_configured() ) {
+			// If the plugin is active, but not configured, the button will send them to the setup wizard.
+			$button = $this->get_link_parameters();
+		}
+
+		return $button;
+	}
+
+	/**
+	 * Gets the array of parameters for the link to configure WP Mail SMTP.
+	 * @todo maybe this should go to the settings page instead.
```

Comment:
@cklosowski should this button go to the setup wizard or to the WPSMTP settings page please? I was looking at their code to see if there was an easy rule to make that determination but not really seeing anything.

The settings page does have a link to the setup wizard so that may make more sense than just launching the wizard. Otherwise, if we are going to launch the wizard then I think the button should say "Launch the WP Mail SMTP Wizard" instead of "Configure..". Just wanted to check with you first.